crates/libtiny_common/src/lib.rs

```diff
@@ -157,6 +161,9 @@ pub enum MsgTarget<'a> {
     /// Show the message in the privmsg tab.
     User { serv: &'a str, nick: &'a str },
 
+    /// Show the message in a miscellaneous tab.
+    Misc { name: &'a str },
```

Comment:
Since we know a `Misc` tab is always one of mentions or help (or "errors" though I'm not sure if that should be a part of this PR, see above), I think we can avoid the string here and use a `MiscTab` enum:

```
enum MiscTab {
    Mentions,
    Help,
    ...
}
```

This makes invalid states like `Misc { name: "wat" }` unrepresentable, and comparisons are also more efficient as we avoid checking string equality and check integer equality.